testament/tester.nim

```diff
@@ -505,7 +539,17 @@ proc main() =
       targetsStr = p.val.string
       targets = parseTargets(targetsStr)
     of "nim": compilerPrefix = p.val.string
-    else: quit Usage
+    of "directory":
+      setCurrentDir(p.val.string)
+    of "colors":
+      if p.val.string == "on":
```

Comment:
You could use ``strutils.parseBool`` here. (Not required, just that you know.) Or a ``case`` statement for strings.